lib/lpm.c

```diff
@@ -60,7 +65,11 @@ lpm_lookup_ipv4(struct rte_lpm *lpm, uint32_t ip)
 		G_LOG(ERR, "lpm: incorrect arguments for IPv4 lookup\n");
 		goto out;
 	} else if (ret == -ENOENT) {
-		G_LOG(WARNING, "lpm: IPv4 lookup miss\n");
+		char buf[INET_ADDRSTRLEN];
+		if (inet_ntop(AF_INET, &ip, buf, sizeof(buf)) != NULL)
+			G_LOG(NOTICE, "lpm: IPv4 lookup miss for %s\n", buf);
+		else
+			G_LOG(NOTICE, "lpm: IPv4 lookup miss\n");
```

Comment:
`lpm_lookup_ipv4()` and its sister `lpm_lookup_ipv6()` are heavily called and, during normal operation, there'll be misses. So let's promote to `DEBUG` log level.

In addition, `inet_ntop()` is not "free", so we should only call it when we are going to log its result.

This comment also applies to the IPv6 case.

Reply:
Are you saying that (1) it's not worth it to call `inet_ntop()` at all here since this call can happen in the data path and knowing the IP address is extraneous, or (2) in the `else` we should at least log what caused `inet_ntop()` to fail?

Reply:
I meant that we should test that `G_LOG()` is going to log it, if so, then call `inet_ntop()`. A perfect test may be too cumbersome, but testing if the current log level is greater or equal to `DEBUG` would be reasonable.

Reply:
Adding the `errno` to the `else` clause is a good idea.